workflows/alevin-quant/run-alevin.nf

```diff
@@ -48,12 +49,14 @@ process alevin{
 }
 
 workflow{
-  sample_ids = params.sample_ids?.tokenize(',') ?: []
-  ch_reads = Channel.fromList(sample_ids)
+  run_ids = params.run_ids?.tokenize(',') ?: []
```

Comment:
Is there where the `id` value come from in `check-md5.nf`?

Reply:
Sort of... this is a super strange thing, actually, and I got it straight from the developer in a chat conversation.

`params.run_ids` is the string defined at the top (or it can be given at the command line). So in this case by default it is "SCPCR000001,SCPCR000001"

`.tokenize(',')` is basically splitting at the comma and turning it into a list (like `str_split(',')`).  

The strange part is the `?` What that does is say that if `params.run_ids` doesn't exist or is blank, then skip the `tokenize` step and just return an empty list. 

I'm then using that list to filter the data table in the metadata file to only the samples I want to actually run.

Reply:
Ah okay, this also makes more sense to me now. I agree the `?` part is strange, but makes sense when you think about it ¯\_(ツ)_/¯